ochazuke/models.py

```diff
@@ -23,6 +23,10 @@ class Milestone(db.Model):
     title = db.Column(postgresql.TEXT, unique=True, nullable=False)
     issues = db.relationship('Issue', backref='milestone', lazy=True)
 
+    # make initializing of milestone objects more concise
```

Comment:
@karlcow I'm totally happy to add more or better explanations in a useful way, but I feel like I'm going to be repeating the model-level docstrings here if I add them to the `__init__ ` methods.

Maybe I'm misunderstanding your request?

Would either of these help? I could try to:
• Do bullets for the `__init__` functions, as a reminder
or
• Format the model-level docstrings better to help them stand out and be more clear/concise